go-server-server/go/default.go

```diff
@@ -20,11 +20,20 @@ const PING_COMMAND_STR string = "ping"
 
 func StateHeartbeatGet(w http.ResponseWriter, r *http.Request) {
     w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+    var availableRoutes int = -1
```

Comment:
CRM stats will be available only after 5 minutes of system start.. so we can update the API to ignore the value if it is returned as "-1"

Reply:
I have updated the contract mentioning this.